### PR TITLE
Tests travis

### PR DESCRIPTION
**Pull request by john-kheir**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,29 @@
-language: go
-sudo: true
-dist: trusty
+matrix:
+  include: 
+    - language: python
+      python: 3.5
+      before_script:
+        - sudo apt-get -qq update
+        - sudo apt-get install -y python-pip
```
Comment (grimpy) [inline .travis.yml]: shouldnt this be python3-pip ?

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
+        - pip install -r tests/requirements.txt
```
Comment (grimpy) [inline .travis.yml]: pip3 ?

File: .travis.yml [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
+        - pip install client/py-client/.
```
Comment (grimpy) [inline .travis.yml]: pip3?

File: tests/config.ini [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[main]
+target_ip=
+zt_access_token=a7djPBPJH6dtBNCw6PnkdR5RRDskwwFv
```
Comment (grimpy) [inline tests/config.ini]: what is this token?

Comment (john-kheir) [inline tests/config.ini]: it is for zerotier network

File: .travis.yml [hunk 5]
```diff
@@ -1,21 +1,28 @@
-language: go
-sudo: true
-dist: trusty
+matrix:
+  include: 
+    - language: python
+      python: 3.5
+      before_script:
+        - sudo apt-get -qq update
```
Comment (grimpy) [inline .travis.yml]: this apt-get update is not need for anything please remove

**Pull request merged**